website/content/docs/nia/tasks.mdx

```diff
@@ -61,6 +61,45 @@ The services condition operates by monitoring the [Health List Nodes For Service
 | `node_tagged_addresses` | List of explicit LAN and WAN IP addresses for the agent                                           |
 | `node_meta`             | List of user-defined metadata key/value pairs for the node                                        |
 
+The services condition is the default behavior if no `condition` block is configured for a task. The two tasks below have equivalent behavior, which is to execute when any of the listed services have changes to the previously stated attributes.
+
+```hcl
+task {
+  name = "services_condition_task_1"
+  description = "execute on changes to services whose name starts with web"
+  providers = ["my-provider"]
+  source = "path/to/services-condition-module"
+  description = "execute on changes to api, db, and web services"
```

Comment:
looks like there's two descriptions for this task. suggestion to replace the description at 69 with this one (line 72) so that this description can be at the top of the task config